package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "codeql",
-  "version": "3.24.1",
+  "version": "3.25.0",
```

The version is bumped during the release process. It shouldn't be changed manually.

```suggestion
  "version": "3.24.1",
```